movie-showcase/src/routes/managing/+page.svelte

```diff
@@ -0,0 +1,128 @@
+<script lang="ts">
+    import { movieArray } from "../../stores";
+    import { slideDuration } from "../../stores";
+    import { carouselItemCount } from "../../stores";
+
+    let title: string;
+    let year: string;
+
+    async function addToArray() {
+        const response = await fetch("/managing", {
+            method: "POST",
+            headers: {
+                "Content-Type": "application/json",
+            },
+            body: JSON.stringify({
+                title: title,
+                year: year,
+            }),
+        });
+
+        const movie = await response.json();
+
+        movieArray.update((arr) => [
+            ...arr,
+            {
+                Title: movie.Title,
+                Year: movie.Year,
+                Plot: movie.Plot,
+                Poster: movie.Poster,
+            },
+        ]);
```

Comment:
When the call to OMDb API on the server runs into an error it still returns a "broken" movie which is then added to the `movieArray` here. There should be some kind of error handling and a way to inform the user of problems with their input.